src/taskQueue/taskQueueModel.js

```diff
@@ -71,6 +72,18 @@ function hasSameState(task1, task2) {
     return false;
 }
 
+function workaroundMessages(taskData) {
+    return taskData.map(item => {
+        if (item.interpolationMessage) {
+            item.message = __(item.interpolationMessage, ...item.interpolationData);
```

Comment:
Just a heads up that `interpolationData` can be an empty array sometimes.

Reply:
Should be fine with empty arrays as long as it's an array and not undefined or null